### PR TITLE
Add Dragon Ball Heroes: Ultimate Mission

### PR DESCRIPTION
**Pull request by LOuroboros**
Adds Dragon Ball Heroes: Ultimate Mission.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/dragon-ball-heroes-ultimate-mission/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Dragon Ball Heroes: Ultimate Mission"
+description = "A port of the japanese card-based fighting arcade game \\\"Dragon Ball Heroes\\\", in which you can experience the different stories of the Dragon Ball series, including movies and spinoffs like Dragon Ball GT."
```
Comment (Leo626) [inline games/dragon-ball-heroes-ultimate-mission/game.dat]: The description should start off as a proper sentence with the game's title - `Dragon Ball Heroes: Ultimate Mission is port of...`.

Comment (Leo626) [inline games/dragon-ball-heroes-ultimate-mission/game.dat]: The j in `japanese` should be capitalized - `Japanese`.

Comment (Leo626) [inline games/dragon-ball-heroes-ultimate-mission/game.dat]: There should only be one backslash in `\\\"Dragon Ball Heroes\\\"` i.e. `\"Dragon Ball Heroes\"`.

Comment (Leo626) [inline games/dragon-ball-heroes-ultimate-mission/game.dat]: `you` sounds a bit more like advertising. Use `the player` instead.

**Comment by CodingKoopa** [comment]
Please address the review comments, or this will be closed.

**Comment by LOuroboros** [comment]
@CodingKoopa https://github.com/citra-emu/citra-games-wiki/pull/395/commits/435bdde5957d8e365a6e06fad5a327b8b02d4436

Did I do the changes correctly? I still don't fully know how GitHub works so I hope I didn't screw this up.

**Pull request merged**